projection/core/src/multi-jvm/scala/com/lightbend/lagom/internal/projection/ProjectionRegistrySpec.scala

```diff
@@ -73,18 +76,24 @@ class ProjectionRegistrySpec
       val tagNamePrefix  = projectionName
       val tagNames       = (1 to 5).map(id => s"$tagNamePrefix-$id")
       val tagName001     = tagNames.head
+      val allButTag001   = tagNames.tail
       val coordinates001 = WorkerCoordinates(projectionName, tagNames.head)
 
       registerProjection(projectionName, tagNames.toSet)
 
-      // await until seen as ready
-      expectWorkerStatus(projectionName, tagName001, Started)
+      // await until all seen as 'Started'
+      expectProjectionStatus(projectionName, 5, Started)
 
       enterBarrier("request-pause-worker-test-all-nodes-ready")
       runOn(RoleName("node2")) {
         projectionRegistry.stopWorker(coordinates001)
       }
       expectWorkerStatus(projectionName, tagName001, Stopped)
+
+      // all other tags should be 'Started'
+      allButTag001.foreach { tag =>
+        expectWorkerStatus(projectionName, tag, Started)
+      }
```

Comment:
Taking another approach here. First checking that all have started and then stopping 001. 

And then checking again that the others didn't stop.